city_scrapers/spiders/il_opioid_response.py

```diff
@@ -0,0 +1,144 @@
+                break
+
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description=self._parse_description(item),
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=self._parse_end(item),
+                all_day=self._parse_all_day(item),
+                time_notes=self._parse_time_notes(item),
+                location=self._parse_location(item),
+                links=self._parse_links(item, response),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        return ""
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        return ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return ADVISORY_COMMITTEE
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        date_match = self._get_date(item)
+        if date_match:
+            start_date = datetime.strptime(date_match.group(0), '%m.%d.%y')
+            return datetime.combine(start_date, time(13))
+        else:
+            return None
+
+    def _parse_end(self, item):
+        """Parse end datetime as a naive datetime object. Added by pipeline if None"""
+        date_match = self._get_date(item)
+        if date_match:
+            end_date = datetime.strptime(date_match.group(0), '%m.%d.%y')
+            return datetime.combine(end_date, time(15))
+        else:
+            return None
+
+    def _parse_time_notes(self, item):
+        """Parse any additional notes on the timing of the meeting"""
+        return ""
+
+    def _parse_all_day(self, item):
+        """Parse or generate all-day status. Defaults to False."""
+        return False
+
+    def _parse_location(self, item):
+        """Parse or generate location."""
+        return [
+            {
+                "address":
+                    "401 S. Clinton Street, 7th Floor Executive Conference Room,"
+                    " Chicago, IL 60607",
+                "name": "Illinois Department of Human Services Clinton Building",
+            },
+            {
+                "address":
+                    "100 S. Grand Avenue East, 3rd Floor Executive Video Conference Room,"
+                    " Springfield, IL 62762",
+                "name": "Illinois Department of Human Services Harris Building",
+            },
+        ]
+
+    def _parse_links(self, item, response):
+        """Parse or generate links for meeting agenda and minutes."""
+        links = []
+        agenda = {
+            "type": "agenda",
```

Comment:
The type attribute can be removed here, since we only include title and href for links. Was it mentioned anywhere in the docs? We’ve changed the setup a bit so if it’s mentioned somewhere I’m happy to remove it to reduce confusion

Reply:
I added the `type` key as a way to differentiate between a link to a meeting agenda vs a link to meeting minutes. The page lists both, is there a better way to do this?

Reply:
Gotcha, thanks for clarifying! If you know for sure which links will be what, it’s fine to just make “Agenda” or “Minutes” the title property if the existing  link text isn’t useful

Reply:
Fair point, that can be determined by the link text.